terraform/transition.fec.gov.tf

```diff
@@ -8,14 +8,20 @@
 # and because you can't CNAME the root domain, Savvis/CenturyLink are 
 # pointing the root at an FEC server that HTTP redirects to www.fec.gov
 
-
 resource "aws_route53_zone" "transition_gov_us_zone" {
   name = "transition.fec.gov"
   tags {
     Project = "dns"
   }
 }
 
+resource "aws_route53_zone" "www_fec_gov" {
```

Comment:
Any reason this doesn't end with _zone a la https://github.com/18F/dns/blob/deploy/terraform/18f.gov.tf#L1 ?